README.md

```diff
@@ -109,35 +109,33 @@ $ CUDA_VISIBLE_DEVICES=0 BuildProducts/bin/<executable>   ### use the device num
 
 In order to include AdePT in a separate project we need to run:
 
-```
+```cmake
 find_package(AdePT)
 ```
 
 Which has the same dependencies as before (VecGeom, VecCore and G4HepEM).
 
-Then, for the targets using AdePT:
+Due to the design of the CUDA interface in the VecGeom dependency, linking to
+AdePT whether or not your project uses CUDA code requires use of CMake wrapper
+functions to ensure the correct device linking. For an executable target:
 
-```
-target_include_directories(example_target <SCOPE> 
-                          <TARGET INCLUDE DIRECTORIES>
-                          ${AdePT_INCLUDE_DIRS})
+```cmake
+include(CudaRdcUtils)
```

Comment:
Could this be done transparently in AdePTConfig.cmake.in?

Reply:
I'd prefer to enforce an explicit include for now for clarity in downstream use It also matches the Celeritas pattern of use. We can of course review (in both) in future.